.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: go
 
 go:
-  - 1.8.x
-  - 1.9.x
-  - 1.10.x
+  # - 1.8.x
+  # - 1.9.x
+  # - 1.10.x
```

Comment:
We disabled this until https://github.com/openzipkin/zipkin-go/pull/113 gets merged.